CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Fixed
```

Comment:
I would say this is `### Changed` as this is not a bug but changing the existing features in a backwards compatible way.

Reply:
Works for me.  Done.